config/initializers/paperclip.rb

```diff
@@ -6,31 +6,31 @@
 
   if Rails.env.test?
     config.paperclip_s3_default_options.merge!({
-      :storage => :filesystem,
-      :path => ':rails_root/tmp/attachments/test/s3/:class/:attachment/:id_partition/:style/:filename'
+      storage: :filesystem,
```

Comment:
I think Hound wants us to indent all the way beyond the parenthesis. If this should come up more often, we have to tweak the Rubocop config.